cmd/manager/cleanup.go

```diff
@@ -0,0 +1,178 @@
+package main
```

Comment:
In general, I would like to state that this seems like the wrong abstraction. It is unclear what is being "cleaned" and "why" we are cleaning. I think that instead that we would be better with a migrator of some kind, if we feel like an abstraction is needed. And that migrator should know about what specific migration is happening, not that just generically we need to "clean up".

Also, have we considered if this code will stick around forever? Or is this something we will remove after openshift 4.2 release? It seems like if (more likely, when) we need to do another such migration it wouldn't make sense to do these checks as well.

Reply:
@kevinrizza what if we moved all of these to a separate package called `migrator`, and document why we need this `migrator` to migrate from a openshift 4.1.1 release to a 4.1.2? (Hope I'm getting the versions correct). 
That way, when we need to do another migration, the whole abstraction can be removed.

Reply:
4.1.x to 4.2.0 is what we are concerned about here.

Reply:
I am +1 on `migrator` too.